tasks/install-from-pypi.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Install latest Certbot from pypi
+  when:
+    - certbot_pypi_version == "latest"
+  pip:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - certbot
+    - certbot-nginx
+
+- name: Install latest Certbot from pypi
+  when:
+    - certbot_pypi_version != "latest"
+  pip:
+    name: "{{ item }}"
+    state: present
+    version: "{{ certbot_pypi_version }}"
+  with_items:
+    - certbot
+    - certbot-nginx
```

Comment:
why is this task duplicated ? Why not just rely on `{{ certbot_pypi_version }}` ?

Reply:
Oh nevermind just saw the `state` differs between both tasks